### ISSUE TITLE
FAQ points to Homebrew un-install script, not Linuxbrew

### ISSUE THREAD

**Issue by dg1727** [Linuxbrew/brew#59]
In document brew/share/doc/homebrew/FAQ.md (in the Linuxbrew repo even though that filepath says "homebrew") the section "How do I uninstall Homebrew?" points (twice) to the Homebrew un-install script as opposed to the Linuxbrew one.  

The Homebrew script contains a couple of things that don't work on Linux (at least not on my Debian-derived flavor).  

Currently, the Homebrew & Linuxbrew versions of the file are identical.  I have made my own version of the script, trying to fix the issues I found.  My attempted fixes may show up on Github soon.  

Maybe there should also be a general (edited, not purely automatic) search-and-replace of "Homebrew" to "Linuxbrew" in the Linuxbrew documentation?  

Thanks for checking into this.

**Comment by sjackman** [comment]
The interpreter of the uninstall script was fixed by PR https://github.com/Linuxbrew/install/pull/2
A PR to fix other issues that you've encountered is welcome.
I've fixed the FAQ. Thanks for the bug report.

**Comment by dg1727** [comment]
You're right, the Homebrew & Linuxbrew versions of the uninstall script are **_not_** identical.  When I first downloaded it, I must have 
1. noticed that the URL pointed to Homebrew & 
2. corrected it to Linuxbrew for my own download without remembering that I did that.  

So when, in writing the bug report, I compared what I had locally as "the version pointed to by the FAQ" with a version downloaded at that moment from the Linuxbrew URL, the 2 files were the same.
